pkg/kvdpa/device.go

```diff
@@ -242,7 +242,7 @@ func extractBusNameAndMgmtDeviceName(fullMgmtDeviceName string) (busName string,
 }
 
 /*
-GetVdpaDevicesByPciAddress returns the VdpaDevice objects for the given pciAddress
+GetVdpaDevicesByPciAddress returns the VdpaDevice devices for the given pciAddress
```

Comment:
is't that redundant? and, kind of inaccurate (assiming we can call golang's structs objects)? Do you like "structs" better? or maybe just ommit the word?